tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -239,13 +230,14 @@ def convert_onnx_models_to_ort():
     else:
         session_options_config_entries["session.qdqisint8allowed"] = "0"
 
-    for optimization_level in args.optimization_level:
-        print(f"Converting models and creating configuration file for optimization level '{optimization_level}'")
-        _convert(model_path_or_dir, optimization_level, args.use_nnapi, args.use_coreml, custom_op_library,
-                 args.save_optimized_onnx_model, args.allow_conversion_failures, args.target_platform,
```

Comment:
It would be nice to preserve this behavior if a specific optimization style is not requested. A new user may not know if they want pre-optimized or runtime optimized, so defaulting to creating both models means they only have to run the script once to have both variants to perf test with.